xmlcommand/BESXMLInterface.cc

```diff
@@ -193,7 +191,8 @@ void BESXMLInterface::build_data_request_plan()
                     // will be deleted by the BESXMLInterface destructor when that iterates through
                     // d_xml_cmd_list. jhrg 5/11/22
                     delete current_cmd;
-                } else {
+                }
```

Comment:
I never know which you want:
```
{
else {
```
or 
```
} else {
```
I feel like this gets flip-flopped a lot...

Reply:
Yeah, I used automatic formatting and it likes '} else {' but I hate it. I'm learning to live with it because gettign what I want with 'else' messes up other stuff I care about more.